Orchestrator/README.md

```diff
@@ -1,34 +1,49 @@
 # Orchestrator (PREVIEW)
 
-Conversational AI applications today are built using disparate technologies to fulfill language understanding (LU) needs e.g. [LUIS][1], [QnA Maker][2]. Often, conversational AI applications are also built by assembling different [skills][3] each of which fulfill a specific conversation topic and can be built using different LU technologies. Hence, conversational AI applications typically require LU arbitration/ decision making to route incoming user request to an appropriate skill or to dispatch to a specific sub-component. Orchestration refers to the ability to perform LU arbitration/ decision making for a conversational AI application.  
+Conversational AI applications today are built using disparate technologies to fulfill language understanding (LU) needs, such as [LUIS][1] and [QnA Maker][2]. Often, conversational AI applications are also built by accessing various skill bots, such as [Virtual Assistant skills][3], each of which handle a specific conversation topic and can be built using different LU technologies. Hence, conversational AI applications typically require LU to route an incoming user request to an appropriate skill or to dispatch to a specific sub-component.
 
-[Orchestrator][18] is a [transformer][4] based solution that is optimized for conversational AI applications. It is built ground-up to run locally with your bot.
+<!-- Coining the term "orchestration" doesn't seem necessary. -->
+
+<!-- Discussing the underlying technology also doesn't seem necessary here. Mention in the technical overview, instead. -->
+
+Orchestrator is an LU solution optimized for conversational AI applications. It is built ground-up to run locally with your bot. See the [technical overview][18] for additional details.
 
 ## Scenarios
-**Dispatch**: Orchestrator is a successor to [dispatch][5]. You can use Orchestrator instead of the current dispatch solution to arbitrate across your [LUIS][1] and [QnA Maker][2] applications. With Orchestrator, you are likely to see:
+
+**Routing**: For bots, Orchestrator can replace the [LUIS Dispatch tool][5]. You can use Orchestrator instead of Dispatch to aggregate multiple [LUIS][1] and [QnA Maker][2] applications. With Orchestrator, you are likely to see:
```

Comment:
They used the term arbitrate intentionally. Dependening on your perspective, it essentially selects the "correct" subsequent service to service the request. It doesn't really combine, rather selects a winner to route to.

Reply:
fixed